buildkite/bazelci.py

```diff
@@ -1036,7 +1036,7 @@ def clone_git_repository(git_repository, platform, git_commit=None):
     execute_command(["git", "submodule", "update", "--init", "--recursive", "--force"])
     execute_command(["git", "submodule", "foreach", "--recursive", "'git reset --hard'"])
     execute_command(["git", "clean", "-fdqx"])
-    execute_command(["git", "submodule", "foreach", "--recursive", "git", "clean", "-fdqx"])
```

Comment:
I don't think you have to quote this - execute_command() calls subprocess.run() which will take care of it.

Also, can you fix the other calls to "git submodule foreach", too? Just because they happen to work doesn't mean we shouldn't fix them, too. :)

Reply:
I'm not sure what's going on with the comments here - I've already fixed the other calls, but will remove the quotation marks.

Reply:
Weird, GitHub didn't show me the diff for the commit that changed the other two calls. Sorry.

All looks good now!

Reply:
I also saw some of your comments only in Gmail, but not here.